site/src/atomics/typography.md

```diff
@@ -77,3 +77,13 @@ The following class is used to alter the letter spacing.
 <p>Text with normal letter spacing</p>
 <p class="letter-spacing-wide">Text with wide letter spacing</p>
 ```
+
+## Styling with gradient
```

Comment:
We don't have a good way to doing this with localized text, do we? Should we be adding a pattern we can't support to the design system?

Reply:
Why would it be a problem with localized text?

Reply:
Removed completely for now